vault/resource_gcp_auth_backend_role_test.go

```diff
@@ -208,7 +209,8 @@ resource "vault_gcp_auth_backend_role" "test" {
     project_id             = "%s"
     ttl                    = 300
     max_ttl                = 600
-    policies               = ["policy_a", "policy_b"]
+	policies               = ["policy_a", "policy_b"]
+	add_group_aliases 	   = true
```

Comment:
Would you be willing to fix the alignment here?

Reply:
Will do. I just realised the entire file is tabbed with `\t` but the fixture is with spaces...

Reply:
Done.